src/geotop/math.optim.h

```diff
@@ -0,0 +1,3 @@
+#include <cmath>
+
+double power(double a, double b);
```

Comment:
implement the function here and not in the `.cc` sucht that it can be inlined
and reimplement it as
```
template <class T>
T power(const T a, const T b){
#ifdef MATH_OPTIM
  return std::exp(b*std::log(a));
#else
  return std::pow(a,b);
#endif
}
```